src/Collections/DeletableObjectList.cs

```diff
@@ -166,7 +171,7 @@ public void ForceCompact()
         /// </summary>
         public void TrimExcess()
```

Comment:
Style nit: I prefer methods calling down. Can we move `TrimExcess` above `ForceCompact`?